mq/util.go

```diff
@@ -78,15 +78,16 @@ func publish(host *models.Host, dest string, msg []byte) error {
 	if encryptErr != nil {
 		return encryptErr
 	}
-	if mqclient == nil || !mqclient.IsConnectionOpen() {
+	if mqclient == nil || !mqclient.IsConnected() {
```

Comment:
won't this be more accurate check with `IsConnectionOpen` ?
otherwise this will cause timeout errors, if the connection is not active

Reply:
Yes, IsConnectionOpen is the func to check if it's connected in deed.   IsConnected includes the scenarios if it's in re-connecting...
let me change it to IsConnectionOpen. But it will impact the publish as well. If it's in re-connecting status, the publish will be dropped.